lib/hooks/moduleloader/index.js

```diff
@@ -117,6 +153,11 @@ module.exports = function(sails) {
       // Expose self as `sails.modules` (for backwards compatibility)
       sails.modules = sails.hooks.moduleloader;
 
```

Comment:
I used an array to build the regexes. So if we need to add another language later, we dont have to go edit EVERY regex in the file...